### PR TITLE
Some rough fixes after the transfer

### PR DESCRIPTION
**Pull request by krasi-georgiev**
Fixed import paths
Use the common makefile
Fixed some lint error.

Also
Fixes: #9 

An updated version of https://github.com/prometheus-community/jiralert/pull/34

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,78 +1,3 @@
...
-	@mv $(GOBIN)/errcheck $(ERRCHECK)
-	@go mod tidy
-
-$(STATICCHECK):
-ifeq ($(GOBIN),)
-	@echo >&2 "GOBIN environment variable is not defined, where to put installed binaries?"; exit 1
-endif
-	@echo ">> getting staticcheck"
-	@GO111MODULE=on GOOS= GOARCH= $(GO) get -u honnef.co/go/tools/cmd/staticcheck
+include Makefile.common
```
Comment (bwplotka) [inline Makefile:3]: wow (:

File: cmd/jiralert/main.go [hunk 2]
```diff
@@ -1,3 +1,16 @@
+// Copyright 2017 The Prometheus Authors
```
Comment (bwplotka) [inline cmd/jiralert/main.go:1]: hmmm It's not prometheus authors. I think we have to put some jiralert authors or something instead :thinking: 
cc @free

**Comment by krasi-georgiev** [comment]
> Hope it's ok with @free

yep I can also add @free  as a contributor to the project so he can also have access to merge PRs.

**Comment by bwplotka** [comment]
I think he is already.

File: cmd/jiralert/main.go (hunk 2 above)
Comment (krasi-georgiev) [reply cmd/jiralert/main.go:1]: I think it is ok since @free is part of the prometheus-community team than we are all part of the Promethues Authors.

**Comment by krasi-georgiev** [comment]
removed the Travis prerequisite and enabled circle.


merging so that @free or anyone else can open additional PRs for improvements and fixes

**Pull request merged**